x-pack/plugin/searchable-snapshots/qa/s3/build.gradle

```diff
@@ -0,0 +1,69 @@
+import org.elasticsearch.gradle.info.BuildParams
+
+import static org.elasticsearch.gradle.PropertyNormalization.IGNORE_VALUE
+
+apply plugin: 'elasticsearch.standalone-rest-test'
+apply plugin: 'elasticsearch.rest-test'
+
+final Project fixture = project(':test:fixtures:s3-fixture')
+final Project repositoryPlugin = project(':plugins:repository-s3')
+
+dependencies {
+  testCompile project(path: xpackModule('searchable-snapshots'), configuration: 'testArtifacts')
+  testCompile repositoryPlugin
+  testCompile fixture
```

Comment:
Does the s3-fixture actually need to be on the test classpath? As far as I can tell there are not references to that code and the fixture is just something running in a Docker container which we talk to over an API.

Reply:
Good catch, I removed this dep